response.md

```diff
@@ -63,6 +63,10 @@ Additionally, [/reverse](reverse.md) queries will have a `distance` parameter, w
 
 ## Notable features
 
+## `coordinates`
+
+At present, all results returned from Mapzen Search are points. Following the [GeoJSON specification](http://geojson.org/geojson-spec.html#positions), these coordinates are in **longitude, latitude** order.
```

Comment:
Oh, that's a good point (har har). While it's possible we will return polygons some day, we almost certainly won't break geojson support. Ok, time to clean things up.